### DISCUSSION TITLE
New Wavetables

### DISCUSSION THREAD

**Discussion by DenBaum** [bangcorrupt/hacktribe#156] [Ideas]
Hi is there an easy way oft getting new wavetables into the electribe? I guess the new synth engines are wavetables. If so maybe I could replace them in the update image if I knew their file offsets and format?

**Comment by bangcorrupt** [comment]
The new oscillators are not wavetables, they're the existing analog modeled oscillators with different parameters set.  I don't know if the sound engine includes a wavetable oscillator, it doesn't appear to be an option on the KingKORG.  The KingKORG does have single cycle waveforms with VPM, if this exists in the e2 sound engine I'll try to enable it some day.

**Reply by DenBaum** [reply]
OK thanks for hacktribe btw, great work :)
Another thing i thought about was that we allready almost have wavetables with the sample player. The edit Encoder lets you have like 100 start Pointe per sample. If one would prepare a sample file with like 100 different waves of a filtersweep by any synth for example this could work if the gate time can be short enough...

**Reply by bangcorrupt** [reply]
#58

**Comment by ptowns** [comment]
Yeah been so busy haven't had much time to join in on the discussion. Just updated #58 with a video of the idea for non instagram people. I messed around with making samples that had for example 8 sounds in one sample using audacity.  You could sort of elektron style parameter lock the sample (in this case start point) per step. I had a very hard time with this because its actually only 64 steps out off 100. Its super hard to work with because it's not even! So let's say for example you use 0.1 second samples, you make a sample that is 10 seconds long. So 64 of the 0.1 second slots can be filled and 36 of them left blank. As if this isn't complicated enough the sounds have to match up with a gate time. Anyways divisable by (or of??) 64 seem to be best as there is only 64 positive spots on the edit knob.

**Reply by ptowns** [reply]
On more thought not only should I do a video showing this when I get some time. Also I have always found the edit knob on samples unless in practice. Especially the 0% 12 o'clock position. But in catching up with this forum I saw #87 and thought you could put the random and sample and hold lfo's to the edit knob with depth to create probability!?!? Another experiment i need to film!
